### PR TITLE
feat: sq model static require instead of dynamic

### PR DESCRIPTION
**Pull request by Grishezz**
needed to play nicely with webpack

### CODE REVIEW

File: packages/sequelize-model/src/sequelizeInit.js [hunk 1]
```diff
@@ -53,115 +31,20 @@ function getPoolConfig() {
...
-}
-
-function convertNullToEmptyArray(fieldInSchema, v) {
-  if (fieldInSchema.isList) {
-    return v === null ? [] : v;
-  }
-  return v;
-}
+sq.sequelize = sequelize;
+sq.Sequelize = Sequelize;
```
Comment (barakd) [inline packages/sequelize-model/src/sequelizeInit.js]: double line

File: test/model/minister-model.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { baseModel } = require('./../../src/base/baseModel');
+const { baseModel } = require('@venncity/sequelize-model/src/base/baseModel');
```
Comment (barakd) [inline test/model/minister-model.js]: isn't the baseModel exported from the entry file?
wouldn't const { baseModel } = require('@venncity/sequelize-model') work?
if not, lets make it work, its not healthy to directly access other module src

File: packages/sequelize-model/src/sequelizeInit.js (hunk 1 above)
Comment (Grishezz) [reply packages/sequelize-model/src/sequelizeInit.js]: it's upper and lower case

File: test/model/minister-model.js (hunk 2 above)
Comment (Grishezz) [reply test/model/minister-model.js]: you are right, I moved the file and webstorm automatically refactored the requires

**Pull request merged**